appveyor.yml

```diff
@@ -1,17 +1,18 @@
 environment:
   matrix:
   - julia_version: 0.7
-  - julia_version: latest
+  - julia_version: 1
```

Comment:
I'd rather use `1.0` to ensure we test that that version for as long as we claim to support it.